ci/dev/watch.ts

```diff
@@ -172,22 +179,23 @@ class Watcher {
     }
   }
 
-  private createBundler(out = "dist"): Bundler {
-    return new Bundler(
-      [
+  private createBundler(out = "dist"): FixMeLater {
+    return new (Parcel as any)({
+      entries: [
         path.join(this.rootPath, "src/browser/register.ts"),
         path.join(this.rootPath, "src/browser/serviceWorker.ts"),
         path.join(this.rootPath, "src/browser/pages/login.ts"),
         path.join(this.rootPath, "src/browser/pages/vscode.ts"),
       ],
-      {
-        outDir: path.join(this.rootPath, out),
-        cacheDir: path.join(this.rootPath, ".cache"),
-        minify: !!process.env.MINIFY,
-        logLevel: 1,
+      cacheDir: path.join(this.rootPath, ".cache"),
+      logLevel: 1,
+      defaultConfig: require.resolve("@parcel/config-default"),
```

Comment:
> yeah, it might be possible to just forego @parcel/config-default, and depend on @parcel/core directly instead of parcel>@parcel/core
— @oxy 

Guessing you meant here? I wonder how we could do that 🤔